Manifest.toml

```diff
@@ -98,9 +122,9 @@ version = "0.1.0"
 
 [[DashCoreComponents]]
 deps = ["DashBase"]
-git-tree-sha1 = "4f4935719119350b1f4c464341ef64743920ea74"
+path = "/home/sglyon/.julia/dev/DashCoreComponents"
```

Comment:
Can you the actual DashCoreComponents package in Julia registry instead of dev package?

Reply:
Good catch -- will fix! Waiting on Dash.jl 0.1.6 to be released and will then make the change and push here

Reply:
fixed!